setup.py

```diff
@@ -61,6 +61,7 @@ def get_extras_require() -> Dict[str, List[str]]:
             "mlflow",
             "mpi4py",
             "mxnet",
+            "nbeval",
```

Comment:
Isn't this a typo? 

ref: https://github.com/computationalmodelling/nbval#installation

```suggestion
            "nbval",
```

Reply:
Thank you!

Reply:
It is! Thank you!